source/includes/_account.md

```diff
@@ -4335,8 +4403,8 @@ curl -v \
     -H "X-Killbill-CreatedBy: demo" \
     -H "X-Killbill-Reason: demo" \
     -H "X-Killbill-Comment: demo" \
-    -d '{ "stateName": "STATE1", "service": "ServiceStateService", "isBlockChange": false, "isBlockEntitlement": false, "isBlockBilling": false, "effectiveDate": "2018-07-17T21:17:28.842Z", "type": "ACCOUNT" }' \
-    "http://127.0.0.1:8080/1.0/kb/accounts/10483c3a-3394-4667-8519-0d849e9a8ec2/block"
+    -d '{ "stateName": "STATE1", "service": "ServiceStateService", "isBlockChange": false, "isBlockEntitlement": false, "isBlockBilling": false }' \
+    "http://127.0.0.1:8080/1.0/kb/accounts/10483c3a-3394-4667-8519-0d849e9a8ec2/block?requestedDate=yyyy-MM-dd"
```

Comment:
Instead of `yyyy-MM-dd`, maybe put a real date?

Reply:
Since `requestedDate` is an optional query parameter, I've removed it totally as we are not specifying any optional parameters in any of the requests. However, please let me know if you'd like to keep it.